server/repository/src/migrations/mod.rs

```diff
@@ -0,0 +1,156 @@
+mod types;
+mod v1_00_04;
```

Comment:
so each version upgrade is a new module? and we will have hundreds of them imported here? is that ok? I don't know of alternatives, and overall this is better than the huge method in 4D - just checking.

Reply:
yep, there would be a lot of imports here, should be ok i hope